wolfCLU/src/certgen/clu_certgen_ed25519.c

```diff
@@ -14,6 +35,11 @@ int make_self_signed_ed25519_certificate(char* keyPath, char* certOut) {
     
     int keyFileSz;
     FILE* keyFile = fopen(keyPath,"rb");
+    if (keyFile == NULL) {
+        printf("unable to open key file %s\n", keyPath);
+        return BAD_FUNC_ARG;
+    }
+
     fseek(keyFile, 0, SEEK_END);
```

Comment:
Unlikely but `fseek` can fail. It has a return value that should be checked. Applies everywhere we use `fseek` in wolfCLU (similar to comment on `ftell`).